src/ResourceManager/Websites/Commands.Websites/Commands.Websites.csproj

```diff
@@ -134,7 +133,6 @@
     <Compile Include="Cmdlets\StartAzureWebApp.cs" />
     <Compile Include="Cmdlets\StopAzureWebApp.cs" />
     <Compile Include="Cmdlets\AppServicePlan\GetAzureAppServicePlan.cs" />
-    <Compile Include="Cmdlets\AppServicePlan\GetAzureAppServicePlanMetrics.cs" />
```

Comment:
You can remove those files now and later just revert the commit to bring them back :)